test/sql/hnsw_vector.sql

```diff
@@ -51,14 +51,17 @@ CREATE INDEX ON large_vector USING lantern_hnsw (v);
 \set ON_ERROR_STOP on
 
 -- Validate that index creation works with a larger number of vectors
+SET client_min_messages=debug5;
```

Comment:
I think this file was changed by accident when rebasing. There should be no changes aside from `SET _lantern_internal.is_test = true;` and `SET _lantern_internal.is_test = false;`, and perhaps `SET client_min_messages=debug5;` (it seems like you deliberately added the last one? unsure what it is for)